webapp-django/crashstats/crashstats/jinja2/crashstats/bugzilla_comment.txt

```diff
@@ -4,7 +4,11 @@ This bug was filed from the Socorro interface and is
 report bp-{{ uuid }}.
 =============================================================
 
-{% if crashing_thread_frames -%}
+{% if java_stack_trace -%}
+Java stack trace:
+
+{{ java_stack_trace }}
```

Comment:
This just uses the string verbatim--it doesn't truncate it. I figured I'd kick that can down the road because it involves figuring out "what's too long?" and "what should we do if it's too long?" This makes the situation better. We can make it more better later.